### PR TITLE
setup: Add a script to setup GUI on servers

### PR DESCRIPTION
**Pull request by lzzy12**
Signed-off-by: lzzy12 <jhashivam2020@gmail.com>

### CODE REVIEW

**Comment by lzzy12** [comment]
@lazyc0derx it has been tested well on Ubuntu 18.04 in Google Cloud and Digital Ocean.

**Comment by lzzy12** [comment]
I just need to run this script and allow tcp ports 5901:5910 and connect via a VNC Viewer software to connect to IP:5901 for eg 59.56.213.1:5901

**Comment by Yash-Garg** [comment]
Nice script @Izzy12

**Comment by akhilnarang** [comment]
Nice, but won't be merging this, this repo is for my personal scripts, and a few for setting up an AOSP build environment.

File: setup/gui/Setup.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+##############################
+# Install Desktop Environment
+##############################
+clear
+echo -e "\n Select the Desktop Environment\n 1. Ubuntu \n 2. Xfce4"
+read -p "Enter Your Choice: " CHOICE
+
+case $CHOICE in
+  1)
+    DE_NAME="Ubuntu"
```
Comment (akhilnarang) [inline setup/gui/Setup.sh]: Also Ubuntu is the distribution, not the DE :P

Comment (Yash-Garg) [reply setup/gui/Setup.sh]: Wew

**Comment by lazyc0derx** [comment]
Ahh?
Ubuntu 18.04 will show grey screen!
And I know too we first have to make computer listen to the ports 5901-5910 depending on what you want , currently rdp is good than ssh since default display manager have conflicts with us and this isn't new these types of hacks is present everywhere from ancient times

**Comment by lzzy12** [comment]
@0xXA dude see, it worked for me so I posted here... Anyways it's PR is closed now.. It didn't work for you, not my problem.. dont argue again on pointless topics

**Pull request closed without merging**